File: qe-selenium-python/Activity13.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/tables")
  
    print(driver.title)
   
    
    print("Title of the Page: "+driver.title)

    rows = driver.find_elements(By.XPATH,"//table/tbody/tr")
    cols = driver.find_elements(By.XPATH, "//table/thead/tr/th")
    print("No of Row in the Table: ", len(rows))
    print("No of Cols in the Table: ", len(cols))

    cellValusOfRow = driver.find_elements(By.XPATH, "//table/tbody/tr[3]/td")

    print("Cell Values in the 5 Row")
    for cellValue in cellValusOfRow:
       print(cellValue.text)
      

    cellValue = driver.find_element(By.XPATH,"//table/tbody/tr[2]/td[2]")

    print(cellValue.text)

File: qe-selenium-java/Activity13.java
package demos;

import java.util.List;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity13 {
public static void main(String[] args) {
    WebDriver driver = new FirefoxDriver();
    driver.get("https://training-support.net/webelements/tables");
    System.out.println("Title of the page: "+driver.getTitle());

    List<WebElement> rows = driver.findElements(By.xpath("//table/tbody/tr"));
    List<WebElement> cols = driver.findElements(By.xpath("//table/thead/tr/th"));
    System.out.println("No of Cols in the Table: "+cols.size());
    System.out.println("No of Rows in the Table: "+rows.size());

    List<WebElement> cellValusOfRow = driver.findElements(By.xpath("//table/tbody/tr[3]/td"));

    for(WebElement cellValue : cellValusOfRow){
        System.out.println(cellValue.getText());
      }

    WebElement cellValue = driver.findElement(By.xpath("//table/tbody/tr[2]/td[2]"));

    System.out.println(cellValue.getText());


    driver.quit();

}
}
